client/src/modules/dashboard/shared/components/Layout.tsx

```diff
@@ -54,16 +54,23 @@ export const Layout = ({
 
   return (
     <div data-cy={dataCy}>
-      <HStack {...rest} data-cy="dashboard-tabs" as="nav" my="2">
+      <HStack
+        {...rest}
+        as="nav"
+        data-cy="dashboard-tabs"
+        my="2"
+        overflowX="auto"
+      >
         {linksWithPermissions
           .filter((link) => !link.requiredPermission || link.hasPermission)
           .map((item) => (
             <LinkButton
-              key={item.link}
-              href={item.link}
               colorScheme={
                 router.pathname.startsWith(item.link) ? 'blue' : 'gray'
               }
+              href={item.link}
+              key={item.link}
+              minWidth="fit-content"
```

Comment:
if we add grid `minwidth` can go

```suggestion
```

Reply:
Btw, I remember `key` attribute needs to have `id` value in it (Oliver linked article about it before), but this one looks different, I wonder how is this different from the usual `key` that needs `id`?

Reply:
Key needs to have unique value, I think link matches that need.

Reply:
Good point, I was tunnel visioned on `id` 😓 👍.